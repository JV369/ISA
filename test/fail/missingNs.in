./feedreader http://www.mocky.io/v2/5bd593a8310000360041db26 -au
