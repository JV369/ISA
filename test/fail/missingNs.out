*** Test RDF ***
Test item RDF
Autor: JV
URL: http://xml.com/pub/2000/08/09/xslt/xslt.html

Test item RDF2
Autor: Hue
URL: http://xml.com/pub/2000/08/09/rdfdb/index.html

